mlibrary/search
# [LIBSEARCH-1008] Update Everything results bento box displays with new language

# Overview
It has been decided to add descriptions for each datastore that clearly and succinctly describe key characteristics of the most sought after materials found in each datastore.

This pull request closes [LIBSEARCH-1008](https://mlit.atlassian.net/browse/LIBSEARCH-1008).

## Anything else?
`BentoboxList` has been rebuilt to reduce code and simplify logic.

## Testing
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Check the [everything](http://localhost:3000/everything) datastore and make a search.
  - Do you see the descriptions?
  - Does anything look broken?
